### DISCUSSION TITLE
Awesome Student Programs!!!

### DISCUSSION THREAD

**Discussion by Sonichigo** [EddieHubCommunity/support#2016] [General] [answered]
## <b>This discussions is created to list all the amazing programs out there for the Students!!</b>

**Comment by jayesh-srivastava** [comment]
Just a small request, can you please elaborate more on the inspire question because that's the only question that is sort of vague. Like how to start in that question, what all necessary things to add. : )

**Reply by Sonichigo** [reply]
Sure will edit it!

**Comment by Sonichigo** [comment, marked as answer]
#### 1. [Microsoft Student Ambassadors](studentambassadors.com)
Just a advice, complete the form entirely, there are some optional fields complete them too!! 

### In the inspire one: 
     1. Start with how you have been inspired by technologies
     2. show your Github profiles and project that you have made,
     3. have those projects created have made an impact,
     4. what all communities or open source program you are part of,
     5. plus also tell why you wanna join the program! and what will you do after joining it! (IMPORTANT)

<b><i>The program result gets out on Quarterly basis and till it is in the review, so you can edit it!</i></b>
